cernopendata/modules/fixtures/data/docs/cms-guide-for-conddb/cms-guide-for-conddb.md

```diff
@@ -2,14 +2,34 @@ This page explains the use of global tags and the condition database with the CM
 
 A Global Tag is a coherent collection of records of additional data needed by the reconstruction and analysis software. The Global Tag is defined for each data-taking period, separately for collision and simulated data.
 
-These records are stored in the condition database. Condition data include non-event-related information (Alignment, Calibration, Temperature, etc.) and parameters for the simulation/reconstruction/analysis software. For 2011-2012 CMS Open Data, the condition data are provided as sqlite files in the `/cvmfs/cms-opendata-conddb.cern.ch/` directory, which is accessible to the CMS Open Data VM.
+These records are stored in the condition database. Condition data include non-event-related information (Alignment, Calibration, Temperature, etc.) and parameters for the simulation/reconstruction/analysis software. For CMS Open Data, the condition data are provided as sqlite files in the `/cvmfs/cms-opendata-conddb.cern.ch/` directory, which is accessible to the CMS Open Data VM.
```

Comment:
Perhaps "accessible to the CMS Open"--> "accessible through the CMS Open".  Maybe it gives a better idea that this is being accessed remotely and helps convey the notion that it would be normal to wait a bit for the caching...  Not super important though...

Reply:
👍